LICENSE.txt

```diff
@@ -35,7 +35,7 @@ Mozilla Public License Version 2.0
     means any form of the work other than Source Code Form.
 
 1.7. "Larger Work"
-    means a work that combines Covered Software with other material, in 
+    means a work that combines Covered Software with other material, in
```

Comment:
This is part of the license text, we can't change it afaik, then we are no longer being compliant to the license. In legal english a work means something else afaiui